releasenotes/notes/removed_deprecation_0.20-17b0ea6a3980eb4f.yaml

```
@@ -0,0 +1,26 @@
+---
+upgrade:
+  - |
+    Code and tests deprecated in 0.20 are removed.
+    :class:`.visualization.QCircuitImage` objects no longer supports
+    `qregs`, `cregs`, `layout`and `global_phase` arguments.
+    
+  - |
+    Code and tests deprecated in 0.20 are removed. In
+    :class:`.transpiler.CouplingMap` :meth:`~subgraph` is removed.
+    :meth:`~reduce` can be used in place of 
+    :meth:`~subgraph`.
```

```
:meth:`~subgraph`
```
wont resolve, as it is being removed. By this PR :)

```suggestion
    The method `subgraph` in :class:`.transpiler.CouplingMap`
    is being removed as it was deprecated in 0.20. The method
    :meth:`~reduce` can be used in its place. It does the same thing, but preserves nodelist order.
```